src/styles/blocks/page.scss

```diff
@@ -0,0 +1,8 @@
+html {
+  font-family: Roboto, sans-serif;
+  color: $main-color;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the color property should be defined in `:root` or in a separate variables file. It's good practice to keep all color definitions in one place for easier maintenance.